nvflare/private/fed/app/client/client_train.py

```diff
@@ -128,7 +130,8 @@ def main(args):
             )
 
             fl_ctx.set_prop(FLContextKey.WORKSPACE_OBJECT, workspace, private=True)
-            client_engine.fire_event(EventType.SYSTEM_BOOTSTRAP, fl_ctx)
+            fl_ctx.set_prop(FLContextKey.ARGS, args, private=True, sticky=True)
+            fl_ctx.set_prop(FLContextKey.SITE_OBJ, federated_client, private=True, sticky=True)
```

Comment:
What's SITE_OBJ? It's very vague. Should be more specific like CLIENT_OBJ.

Reply:
server side will have the same for this. Keep it as SITE_OBJ